src/structures/MessageReaction.js

```diff
@@ -9,11 +9,6 @@ const Util = require('../util/Util');
  * Represents a reaction to a message.
  */
 class MessageReaction {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APIReaction} data The data for the message reaction
```

Comment:
This was the last instance of `APIReaction`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/MessageReaction.js#L134-L137